README.md

```diff
@@ -59,15 +56,28 @@ pyenv install 3.8.10
 # make it available globally
 pyenv global system 3.8.10
 ```
+</details>
+
 
-### Poetry
+## Poetry
 
 This project uses `poetry` to install and manage dependencies. To install poetry,
 follow these [instructions](https://python-poetry.org/docs/master/#installation).
 
+# Installation
+
+With `pip`:
+
+```shell
+python3.8 -m pip install volttron-lib-fake-driver
```

Comment:
Shouldn't this be volttron-bacnet-proxy?

Reply:
I've reformatted and revised the README to follow the README pattern set in volttron-platform-driver.